webpack.config.client.js

```diff
@@ -1,6 +1,13 @@
 /* eslint-disable global-require */
 const AssetsPlugin = require('assets-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const dotenv = require('dotenv');
+
+const result = dotenv.config();
```

Comment:
Can you add a comment explaining what we're doing here? Ideally any new starter can look at any part of our Webpack config and it'll be annotated for them.